tasks/lib/uglify.js

```diff
@@ -94,7 +94,7 @@ exports.init = function(grunt) {
           minifyOptions.mangle.properties.reserved = [];
         }
         if (options.reserveDOMProperties) {
-          require('uglify-js/tools/domprops').forEach(function(name) {
+          require('uglify-es/tools/domprops').forEach(function(name) {
```

Comment:
Will do :+1: 

It's not like this `require()` is particularly compute-intensive anyway.